### DISCUSSION TITLE
Reneder Canvas dark bing background map

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#4175] [Q&A]
Hi,
I am trying to display canvas dark bing mackground map, but does not seem to work, my code: 
![image](https://user-images.githubusercontent.com/53996840/186828647-38fba3a3-9720-43b2-a733-dac9a2bc441d.png)

I also tried this: 
![image](https://user-images.githubusercontent.com/53996840/186903622-2026e283-3d8f-43c1-9aed-439b4c44da5a.png)

but none works
Please let me know what I need to do to make it work

**Comment by pmconne** [comment]
Define "not work" please. Do no tiles display? If so are there errors in your browser console? Or, do the normal (non-dark) tiles display? Or, something else?

**Reply by Rahul-0108** [reply]
no tiles display, screen is white completely, the tile network calls:
![image](https://user-images.githubusercontent.com/53996840/186908783-d70209e4-9996-4efc-9c65-d8a2e404e1f2.png)

**Comment by pmconne** [comment]
400 = bad request. @mdastous-bentley can you tell if what @Rahul-0108 is doing can be expected to work?

**Reply by mdastous-bentley** [reply]
'TileUrl' format was designed to work with "XYZ style" tiles servers, thus only **level**, **column** and **row** runtime parameters are supported, such as: https://b.tile.openstreetmap.org/{level}/{column}/{row}.png.

**Reply by Rahul-0108** [reply]
so currentley canvas dark , canvas light are not supported right ?? @mdastous-bentley

**Reply by mdastous-bentley** [reply]
Correct, only 'Road', 'Aerial' and 'AerialWithLabels' are currently supported.
